ModuleConfig.cfc

```diff
@@ -12,115 +12,67 @@ component {
 	this.webURL                     = "http://www.ortussolutions.com";
 	this.description                = "This module provides server-side validation to ColdBox applications";
 	this.version                    = "@build.version@+@build.number@";
-	// If true, looks for views in the parent first, if not found, then in the module. Else vice-versa
-	this.viewParentLookup           = true;
-	// If true, looks for layouts in the parent first, if not found, then in module. Else vice-versa
-	this.layoutParentLookup         = true;
+
 	// Model Namespace
 	this.modelNamespace             = "cbvalidation";
 	// CF Mapping
 	this.cfmapping                  = "cbvalidation";
-	// Auto-map models
-	this.autoMapModels              = true;
-	// Module Dependencies That Must Be Loaded First, use internal names or aliases
+	// Helpers
+	this.applicationHelper          = [ "helpers/Mixins.cfm" ];
+    // Module Dependencies That Must Be Loaded First, use internal names or aliases
 	this.dependencies               = [ "cbi18n" ];
 	// ColdBox Static path to validation manager
 	this.COLDBOX_VALIDATION_MANAGER = "cbvalidation.models.ValidationManager";
+	this.CBVALIDATION_DEFAULT_RESOURCE = "cbvalidation";
+	this.CBVALIDATION_CUSTOM_RESOURCE = "cbvalidationCustom";
 
 	/**
 	 * Configure module
 	 */
 	function configure(){
-		// Mixin our own methods on handlers, interceptors and views via the ColdBox UDF Library File setting
-		arrayAppend(
-			controller.getSetting( "ApplicationHelper" ),
-			"#moduleMapping#/models/Mixins.cfm"
-		);
-	}
+
+        settings = {
+            manager = this.COLDBOX_VALIDATION_MANAGER,
+            i18nResource = "",
+            sharedConstraints = {
+            }
+        };
+        cbi18n = {
+            resourceBundles = {
+                "#this.CBVALIDATION_DEFAULT_RESOURCE#" = "#moduleMapping#/includes/cbi18n/cbvalidation"
+            }
+        };
+    }
 
 	/**
 	 * Fired when the module is registered and activated.
 	 */
 	function onLoad(){
-		var configSettings = controller.getConfigSettings();
-		// parse parent settings
-		parseParentSettings();
-		// Did you change the validation manager?
-		if ( configSettings.validation.manager != this.COLDBOX_VALIDATION_MANAGER ) {
+        if (len( variables.settings.i18nResource )) {
+            cbi18n.resourceBundles[this.CBVALIDATION_CUSTOM_RESOURCE] = variables.settings.i18nResource;
+        }
+        variables.settings["CBVALIDATION_DEFAULT_RESOURCE"] = this.CBVALIDATION_DEFAULT_RESOURCE;
```

Comment:
You need the names of these resources in validationResults, and instead of hardcoding them I added them to settings.
But since users cannot control the names it is easier to hardcode this